_source/media/media_mirroring.rst

```diff
@@ -0,0 +1,62 @@
+Media Mirroring
+===============
+
+
+Mirroring media uploaded to CollectiveAccess via FTP
+----------------------------------------------------
+
+All media uploaded to CollectiveAccess are stored locally in web-accessible locations defined in the media_volumes.conf configuration file. This is usually all you need, but there are a few cases where being able to automatically copy – or mirror - media to another server can come in handy:
+
+You want to manage audio or video files in CollectiveAccess locally but stream them from a dedicated audio/video host externally, to save local bandwidth or take advantage of features only streaming hosts can provide.
+
+You want to provide high-availability to media files using multiple servers.
+
+You want to make real-time backups of your media.
+
+CollectiveAccess supports mirroring via FTP ("File Transfer Protocol"). FTP is a standard for moving files between servers on the Internet. When FTP mirroring is enabled, every time you upload a new media file the file and its derivatives will be automatically copied using FTP to one or more remote servers. The copying happens in the background so you need not wait for the file transfers to complete before continuing your work. Files of any size (subject to disk space limitations of the mirror servers, of course) may be mirrored.
```

Comment:
This all correct. However the media mirroring system is **very** old. It was actually part of the 0.5x code base and was originally intended to move files to Adobe Flash Streaming Server... that gives you an idea how old it is :-)

At the time everyone still used FTP to move files. Now, you'd never do that. So we should add a note about that somewhere – arguably it's obsolete and should be removed. 

Adding SFTP support would not be hard, so perhaps that should be done. It hasn't to date only because there's been no demand, and so no time spent on it. Until a decision is made, this documentation is ok but should have a note about its status.

(I'm happy to add those note after we merge these changes.)